### PR TITLE
[AAE-10594] Run unit tests in headless mode

### PR DESCRIPTION
**Pull request by nikita-web-ua**
**Please check if the PR fulfills these requirements**

> - [X] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [X] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/AAE-10594

**What is the new behaviour?**

Unit tests run in headless mode

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: demo-shell/karma.conf.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ module.exports = function (config) {
         colors: true,
         logLevel: config.LOG_INFO,
         autoWatch: true,
-        browsers: ['Chrome'],
+        browsers: [/*'Chrome',*/ 'ChromeHeadless'],
```
Comment (mauriziovitale) [inline demo-shell/karma.conf.js:53]: please do not leave code comment out just remove it

**Review by mauriziovitale — CHANGES_REQUESTED** [review]
check comment

File: demo-shell/karma.conf.js (hunk 1 above)
Comment (TomikaArome) [inline demo-shell/karma.conf.js:53]: Why the comment?

Comment (nikita-web-ua) [reply demo-shell/karma.conf.js]: Removed, just followed the style from `process-services/karma.config.js`

Comment (nikita-web-ua) [reply demo-shell/karma.conf.js]: Removed comments

**Review by TomikaArome — APPROVED** [review]

**Review by MichalKinas — APPROVED** [review]

**Comment by mauriziovitale** [comment]
@nikita-web-ua there are unit test failing, not sure if is because they cannot run in headless mode, we should investigate

**Review by AleksanderSklorz — APPROVED** [review]

**Comment by nikita-web-ua** [comment]
@mauriziovitale fixed failing test

**Review by DenysVuika — APPROVED** [review]

**Comment by DenysVuika** [comment]
@mauriziovitale can we merge this PR, any concerns?

**Review by eromano — APPROVED** [review]

**Review by mauriziovitale — APPROVED** [review]

**Pull request merged**